topics/cuda/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Nvidia
+display_name: CUDA
+logo: cuda.png
+released: June 23, 2007
+short_description: CUDA is a parallel computing platform and programming model for NVIDIA GPUs.
+topic: cuda
+url: https://developer.nvidia.com/cuda-zone
+related: nvcc
+wikipedia_url: https://en.wikipedia.org/wiki/CUDA
+---
+CUDA® is a parallel computing platform and programming model developed by NVIDIA for general computing on graphical processing units (GPUs). With CUDA, developers are able to dramatically speed up computing applications by harnessing the power of GPUs.
```

Comment:
Got the blurb from [here](https://developer.nvidia.com/cuda-zone). Do I need to do anything for licensing, or is it fine as-is?

Reply:
Should be OK as is! Thanks for the contribution!